server/src/test/java/org/elasticsearch/common/logging/JsonThrowablePatternConverterTests.java

```diff
@@ -37,25 +37,22 @@ public void testNoStacktrace() throws IOException {
 
     public void testStacktraceWithJson() throws IOException {
 
-        String json = "{"
-            + LINE_SEPARATOR
-            + "  \"terms\" : {"
-            + LINE_SEPARATOR
-            + "    \"user\" : ["
-            + LINE_SEPARATOR
-            + "      \"u1\","
-            + LINE_SEPARATOR
-            + "      \"u2\","
-            + LINE_SEPARATOR
-            + "      \"u3\""
-            + LINE_SEPARATOR
-            + "    ],"
-            + LINE_SEPARATOR
-            + "    \"boost\" : 1.0"
-            + LINE_SEPARATOR
-            + "  }"
-            + LINE_SEPARATOR
-            + "}";
+        String json = """
+            {%s  \
+            "terms" : {%s \
+            "user" : [%s "u1",%s "u2",%s "u3"%s ], %s \
+            "boost" : 1.0%s  \
+            }%s}""".formatted(
+            LINE_SEPARATOR,
+            LINE_SEPARATOR,
+            LINE_SEPARATOR,
+            LINE_SEPARATOR,
+            LINE_SEPARATOR,
+            LINE_SEPARATOR,
+            LINE_SEPARATOR,
+            LINE_SEPARATOR,
+            LINE_SEPARATOR
+        );
```

Comment:
The original code is bizarre. We can make it simpler:

```suggestion
        String json = """
            {
              "terms": {
                "user": [
                  "u1",
                  "u2",
                  "u3"
                ],
                "boost": 1.0
              }
            }
            """;
```